src/recoil/umap.ts

```diff
@@ -1,13 +1,14 @@
 import { atom, selector } from "recoil";
 import { geneDataState } from "./genes";
 import { UMAP } from "umap-js";
-
+import { PCA } from "ml-pca";
 import { Vectors } from "umap-js/dist/umap";
 import { currentCelltypesState, dagDataStructureState } from ".";
+import { diffexpGenesDotplotState } from "./dotplot";
 
 export const umapThresholdState = atom({
   key: "umapThresholdState",
-  default: 200,
+  default: 20000,
```

Comment:
I had changed this to 20,000 so I could create UMAPs on any subset, since I didn't know what node I could click on that would allow UMAP ahead of time.

Do we still want a threshold here?